platform/mds/mds-web/src/main/resources/webapp/js/controllers.js

```diff
@@ -4444,7 +4444,7 @@
             var i, field;
             for (i = 0; i < $scope.allEntityFields.length; i += 1) {
                 field = $scope.allEntityFields[i];
-                if (field.basic.name === name) {
+                if (field.basic !== undefined && field.basic.name === name) {
```

Comment:
@ngraczewski 
If field.basic is undefined and you try to call a property of it, javascript will throw an error and not evaluate the expression (or the rest of the function)

Yea, I know it looks stupid...